src/travis2docker/travis2docker.py

```diff
@@ -221,6 +223,15 @@ def compute_build_scripts(self, prefix_build):
         self.chmod_execution(build_path)
         self.chmod_execution(run_path)
 
+    def _python_version_env(self):
+        versions = self.yml.pop('python', {})
+        if not versions:
+            return False
```

Comment:
What about `2.7` by default?

Reply:
@moylop260 
When in the .travis.yml only have a another version different to "2.7", the travis always generated the "2.7" version

For example
In the .travis.yml only has a "3.2" and the travis2docker generated files to version "2.7" and "3.2"

![image](https://user-images.githubusercontent.com/1387970/30718635-8400ce36-9eee-11e7-805e-d671e2165518.png)